alicloud/resource_alicloud_cdn_domain.go

```diff
@@ -429,6 +467,22 @@ func resourceAlicloudCdnDomainRead(d *schema.ResourceData, meta interface{}) err
 		d.Set("parameter_filter_config", config)
 	}
 
+	if _, ok := d.GetOk("https_config"); ok {
+		ov := d.Get("https_config")
+		oldConfig := ov.(*schema.Set)
+		config := make([]map[string]interface{}, 1)
+		config[0] = map[string]interface{}{
+			"server_certificate":        domain.ServerCertificate,
+			"server_certificate_status": domain.ServerCertificateStatus,
+		}
+		if oldConfig != nil && oldConfig.Len() > 0 {
```

Comment:
I think condition `if oldConfig != nil && oldConfig.Len() > 0` is redundant. private_key is sensitive and it is no need to return.

Reply:
I didn't want to put `private_key` into the state, but I find that if I don't write it then terraform always says it changed. Is there a way to not write `private_key` but also not have terraform keep thinking there are new changes?

Reply:
you can set private_key "Compute: true"

Reply:
@xiaozhu36 maybe it's easier if you change this PR so it works the way you suggest? You should be able to push changes. When I added `Compute: true` and removed `private_key` from the state terraform thinks the resource has changed every run. If I'm doing something wrong it's probably easier for you to push a fix since you're more experienced than I am.